src/s_inter.c

```diff
@@ -114,6 +112,7 @@ struct _instanceinter
     int i_maxfd;
     int i_guisock;
     t_socketreceiver *i_socketreceiver;
+    unsigned char *i_recvbuf;
```

Comment:
why can't this be a fixed-size buffer?

~~~
unsigned char i_recvbuf[NET_MAXPACKETSIZE];
~~~

since `t_pdinstance.pd_inter` is allocated on the heap anyhow, i don't see much use for the additional indirection.

Reply:
good point.